src/main/java/liqp/TemplateParser.java

```diff
@@ -13,24 +13,54 @@
  * The new main entrance point of this library.
  */
 public class TemplateParser {
+
+    public static final TemplateParser DEFAULT_JEKYLL = Flavor.JEKYLL.defaultParser();
+    public static final TemplateParser DEFAULT_LIQUID = Flavor.LIQUID.defaultParser();
+    public static final TemplateParser DEFAULT_LIQP = Flavor.LIQP.defaultParser();
     /**
-     * Returns a {@link TemplateParser} configured with all default settings for "Liquid" flavor.
+     * Returns a {@link TemplateParser} configured with all default settings for "Liqp" flavor.
      */
-    public static final TemplateParser DEFAULT = Flavor.LIQUID.defaultParser();
-    public static final TemplateParser DEFAULT_JEKYLL = Flavor.JEKYLL.defaultParser();
+    public static final TemplateParser DEFAULT = DEFAULT_LIQP;
+
+    /**
+     * Equivalent of
+     * <code>
+     * Liquid::Template.error_mode = :strict # Raises a SyntaxError when invalid syntax is used
+     * Liquid::Template.error_mode = :warn # Adds strict errors to template.errors but continues as normal
+     * Liquid::Template.error_mode = :lax # The default mode, accepts almost anything.
+     * </code>
+     * where usage is like:
+     * <code>
+     *     template = Liquid::Template.parse('', error_mode: :warn )
+     * </code>
+     */
+    public enum ErrorMode {
+        strict,
```

Comment:
Since they're enum constants, they should be all-uppercase (STRICT, WARN, LAX)

Reply:
Well, I simply wanted to follow ruby, so I treat ruby literal as a lower-case enum. Still, changing this probably makes more sense.

Reply:
Yes. I'm hoping we can add some automated checks with PMD, Checkstyle, Spotbugs, etc. They're really helpful. That would come with either adhering more to the Java standard or adding a bunch of exceptions. I'd lean on the former, with very few exceptions only.